index.bs

```diff
@@ -138,10 +138,11 @@ server to declare an embedder policy for a given document. It is a [=Structured
 value MUST be a [=structured header/token=]. [[!I-D.ietf-httpbis-header-structure]] Its ABNF is:
 
 ```
-Cross-Origin-Embedder-Policy = sh-token
+Cross-Origin-Embedder-Policy = sh-item
 ```
 
-The only currently valid `Cross-Origin-Embedder-Policy` value is "`require-corp`".
+The `Cross-Origin-Embedder-Policy` value consists of one token (either "`require-corp`" or
+"`unsafe-none`") which may have a parameter specifying an endpoint for violation reporting.
```

Comment:
Should the value type of the parameter be specified?

https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-2 suggests yes, but the type is not passed along the parsing algorithm, as far as I can tell. @mnot how is this intended to work?

Reply:
It does seem like we should specify the type. Perhaps "specifying a [=structured-header/string=] which represents the endpoint for ..."?

For parsing, I think that the structured headers spec parses parameter values as `sh-item`. I suppose we're on our own to validate them.

Reply:
Done. I also added one more check to the parsing logic.